extension/src/popup/components/account/AccountHeader/index.tsx

```diff
@@ -16,23 +16,20 @@ const ImportedTagEl = () => (
 );
 
 interface AccountHeaderProps {
-  accountDropDownRef: React.Ref<HTMLDivElement>;
+  accountDropDownRef: React.RefObject<HTMLDivElement>;
   allAccounts: Array<Account>;
   currentAccountName: string;
-  isDropdownOpen: boolean;
-  setIsDropdownOpen: (state: boolean) => void;
   publicKey: string;
 }
 
 export const AccountHeader = ({
   accountDropDownRef,
   allAccounts,
   currentAccountName,
-  isDropdownOpen,
-  setIsDropdownOpen,
   publicKey,
 }: AccountHeaderProps) => {
   const { isTestnet } = useSelector(settingsNetworkDetailsSelector);
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
```

Comment:
this didn't need to be passed anymore since we handle all the logic inside this view